packages/components/app/styles/components/dropdown.scss

```diff
@@ -108,6 +109,34 @@ $hds-dropdown-toggle-border-width: 1px;
   }
 }
 
+.hds-dropdown-toggle-icon {
+  border-color: lightgray;
+  padding: 1px;
+}
+.hds-dropdown-toggle-icon__wrapper {
+  align-items: center;
+  border-radius: 3px; // $hds-dropdown-toggle-border-radius- 1px padding - 1px border
+  display: flex;
+  justify-content: center;
+  height: 32px;
+  width: 32px;
+
+  > * {
```

Comment:
[nitpick] this should target only the image. there may be race conditions in which the icon fills one of the corners and since this would apply also to the SVG, it would trim/cut part of the icon